manifests/backup/mysqldump.pp

```diff
@@ -24,11 +26,18 @@
     require       => Class['mysql::server::root_password'],
   }
 
+
+  if $include_triggers {
+    $privs = [ 'SELECT', 'RELOAD', 'LOCK TABLES', 'SHOW VIEW', 'PROCESS', 'TRIGGER' ]
+  } else {
+    $privs = [ 'SELECT', 'RELOAD', 'LOCK TABLES', 'SHOW VIEW', 'PROCESS' ]
+  }
+
```

Comment:
as always, i would prefer you use [selectors](http://docs.puppetlabs.com/puppet/3.8/reference/lang_conditional.html#selectors) here, although, someone else here might disagree ;)